### PR TITLE
Update 5.0.md to include installed apps for Query change

### PR DESCRIPTION
**Pull request by RandomJo**
I was upgrading to Wagtail 5 and got stuck temporarily because I was not able to run the migration command. I figured it out that I needed to add the new app, but this may help future people, especially beginners, not waste time trying to figure out the missing piece.

### CODE REVIEW

File: docs/releases/5.0.md [hunk 1]
```diff
@@ -279,6 +279,8 @@ The undocumented `Page.get_static_site_paths` method (which returns a generator
 The `wagtailsearch.Query` model has been moved from the `search` application to the contrib application `wagtail.contrib.search_promotions`.
 All imports will need to be updated and migrations will need to be run via a management command, some imports will still work with a warning until a future version.
 
+To continue using the `Query` model, you must also add the `wagtail.contrib.search_promotes` application to your project's `INSTALLED_APPS` setting.
```
Comment (gasman) [inline docs/releases/5.0.md:282]: This should be `search_promotions`, not `search_promotes`:
```suggestion
To continue using the `Query` model, you must also add the `wagtail.contrib.search_promotions` application to your project's `INSTALLED_APPS` setting.
```

**Comment by lb-** [comment]
Thank you @RandomJo - this will be helpful alongside https://github.com/wagtail/wagtail/pull/11712

I have added you to the contributors list, let me know if you wanted a different name listed though.

**Comment by lb-** [comment]
Added to other stable branches:

- 5.2.x - aa960d38e661c8119bd523b50d7b2f23d28b3947
- 6.0.x - 948bb19c50955e36a4a72f74c27bb12eacad531b

**Comment by RandomJo** [comment]
No problem, thanks for the quick merge!

**Pull request merged**